src/assets/_project/_blocks/utils/qg-misc.js

```diff
@@ -71,8 +71,9 @@
     },
   ];
   deprecationWarnings.forEach(({selector, label}) => {
-    if ($(selector).length > 0) {
-      console.info(`Please change the font awesome element in ${label} from i to span, we'll be removing the css in this element before 22nd june 2022. Please refer to the https://github.com/qld-gov-au/qg-web-template/pull/391 for more details.`);
-    }
+    // use `show` if the element is created on the fly
+    $(selector).show(() => {
```

Comment:
'show' maybe not be the best function for this functionality. 
I think just checking the length of the element would work?

Reply:
I was simply check the length before, but it doesn't working in scenerio if the call-out-box/alert element is created after the check (which is the reason why the error doen't show up in https://oss-uat.clients.squiz.net/health/staying-healthy/food-pantry/food-labelling/label-buster/_nocache)

yeah, `show` seems doesn't work in this scenerio, i'll have to find another method to listen the creation of the call-out-box/alert element otherwise i'll just use timeout